appinventor/components/src/com/google/appinventor/components/runtime/ChatBot.java

```diff
@@ -1,5 +1,5 @@
 // -*- mode: java; c-basic-offset: 2; -*-
-// Copyright 2023 MIT, All rights reserved
+// Copyright 2023, 2025 MIT, All rights reserved
```

Comment:
We also made changes in 2024 but the copyright was never updated, so we can just do 2023-2025 here.

Reply:
Fixed